project-base/app/assets/js/admin/search/search.js

```diff
@@ -0,0 +1,111 @@
+import Ajax from '@shopsys/framework/js/common/utils/Ajax';
+import Register from 'framework/common/utils/Register';
+
+export default class Search {
```

Comment:
this javascript should be placed in the `packages/framework/assets/js/admin/search` folder